src/modules/75-cf/pages/onboarding/OnboardingDetailPage.tsx

```diff
@@ -111,116 +112,145 @@ export const OnboardingDetailPage: React.FC = () => {
-        spacing="xsmall"
-        flex
-        data-testid="getStartedProgressStepper"
-        height="10px"
-        padding={{ bottom: 'xlarge' }}
-        className={css.keepOnTop}
+        className={css.iconContainer}
+        height="0"
+        width="100%"
+        padding={{ top: 'huge', left: 'xxlarge', right: 'xxxlarge' }}
       >
-        <MultiStepProgressIndicator
-          progressMap={
-            new Map([
-              [0, { StepStatus: firstStepStatus }],
-              [1, { StepStatus: secondStepStatus }],
-              [2, { StepStatus: thirdStepStatus }]
-            ])
-          }
-        />
-      </Layout.Horizontal>
-      <Container className={css.onboardingContainer} height="100%" padding={{ top: 'xxlarge', bottom: 'xlarge' }}>
-        {selectedStep !== STEP.VALIDATE_FLAG ? (
-          <>
-            <Heading level={3} font={{ variation: FontVariation.H3 }} margin={{ bottom: 'large' }}>
-              {getString('cf.onboarding.letsGetStarted')}
-            </Heading>
-            <Heading level={4} font={{ variation: FontVariation.H4 }} margin={{ bottom: 'medium' }}>
-              {getString('cf.onboarding.createFlag')}
-            </Heading>
-
-            {selectedFlag && selectedStep === STEP.SELECT_ENV_SDK && (
-              <Layout.Vertical spacing="medium">
-                <OnboardingSelectedFlag selectedFlag={selectedFlag} />
-                <Divider />
-              </Layout.Vertical>
-            )}
-          </>
-        ) : (
-          <Heading level={3} font={{ variation: FontVariation.H3 }} margin={{ bottom: 'large' }}>
-            {getString('cf.onboarding.validatingYourFlag')}
-          </Heading>
-        )}
-
-        {selectedStep === STEP.CREATE_A_FLAG && (
-          <CreateAFlagView selectedFlag={selectedFlag} setSelectedFlag={setSelectedFlag} />
-        )}
-
-        {selectedStep === STEP.SELECT_ENV_SDK && selectedFlag && (
-          <SetUpYourApplicationView
-            flagInfo={selectedFlag}
-            language={language}
-            setLanguage={setLanguage}
-            apiKey={apiKey}
-            setApiKey={setApiKey}
-            selectedEnvironment={selectedEnvironment}
-            setSelectedEnvironment={setSelectedEnvironment}
-          />
-        )}
-
-        {selectedStep === STEP.VALIDATE_FLAG && selectedFlag && selectedEnvironment && (
-          <ValidateYourFlagView
-            flagInfo={selectedFlag as Feature}
-            language={language as PlatformEntry}
-            apiKey={apiKey as ApiKey}
-            testDone={testDone}
-            setTestDone={setTestDone}
-            environmentIdentifier={selectedEnvironment.identifier}
-          />
-        )}
-      </Container>
-      <Divider />
-      <Layout.Horizontal spacing="small" padding="small">
+        <img src={featureFlagsImg} width="175px" height="50px" />
         <Button
-          text={getString('back')}
-          icon="chevron-left"
+          minimal
```

Comment:
Instead of setting `minimal` and `withoutBoxShadow`, you might get better results using the `ICON` button variation. You also probably don't need the `width` and `height`, but have a play around and see.
```suggestion
          variation={ButtonVariation.ICON}
```

Reply:
This works lovely ^, thank you!